fe/fe-core/src/main/java/com/starrocks/persist/EditLog.java

```diff
@@ -95,53 +90,12 @@ public class EditLog {
     public static final Logger LOG = LogManager.getLogger(EditLog.class);
     private static final int OUTPUT_BUFFER_INIT_SIZE = 128;
 
-    private final Journal journal;
-
     private BlockingQueue<JournalTask> journalQueue;
 
-    @VisibleForTesting
-    public EditLog(Journal journal, BlockingQueue<JournalTask> journalQueue) {
-        this.journal = journal;
+    public EditLog(BlockingQueue<JournalTask> journalQueue) {
         this.journalQueue = journalQueue;
     }
 
-    public EditLog(String nodeName) {
-        journal = JournalFactory.create(nodeName);
-        journalQueue = new ArrayBlockingQueue<JournalTask>(Config.metadata_journal_queue_size);
-    }
-
-    public BlockingQueue<JournalTask> getJournalQueue() {
-        return journalQueue;
-    }
-
-    public long getMaxJournalId() {
-        return journal.getMaxJournalId();
-    }
-
-    public JournalCursor read(long fromId, long toId) {
-        return journal.read(fromId, toId);
-    }
-
-    public long getFinalizedJournalId() {
-        return journal.getFinalizedJournalId();
-    }
-
-    public void deleteJournals(long deleteToJournalId) {
-        journal.deleteJournals(deleteToJournalId);
-    }
-
-    public List<Long> getDatabaseNames() {
-        return journal.getDatabaseNames();
-    }
-
-    public synchronized int getNumEditStreams() {
-        return journal == null ? 0 : 1;
-    }
-
-    public Journal getJournal() {
-        return journal;
-    }
-
     public static void loadJournal(GlobalStateMgr globalStateMgr, JournalEntity journal) {
```

Comment:
Through the confusion was not left( in the original codes, journal was also the data member of class EditLog), still I strongly suggest that change the argument's name.

Reply:
I thought about it before the design review, but there is a lot to involve. We definitely will go through all the logXX interfaces when we will change the phrase of writing journal from after applying to memory to before. Maybe we can refactor this then.